lib/ansible/modules/cloud/vmware/vmware_vm_shell.py

```diff
@@ -16,10 +16,10 @@
 DOCUMENTATION = '''
 ---
 module: vmware_vm_shell
-short_description: Execute a process in VM
+short_description: Starts a program in the guest operating system
```

Comment:
Starts a program is very specific. Why not just:

> Run commands in a VMware guest operating system

Reply:
Done.